stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -1068,13 +1085,15 @@ internal class StripePaymentController internal constructor(
             stripeAccount: String?,
             returnUrl: String? = null,
             enableLogging: Boolean = false,
-            shouldCancelSource: Boolean = false
+            shouldCancelSource: Boolean = false,
+            cancelOnUserNavigation: Boolean = true
```

Comment:
names are hard. definitely open to other ideas for this flag

Reply:
maybe `shouldCancelIntentOnUserNavigation` to be more explicit and consistent?

Reply:
yeah, good call on consistency. I don't really like that this implies we're doing something to cancel the intent (rather than just returning Result.Cancelled) but I guess it's fine